src/lib/QtWidgets/QTreeWidget.ts

```diff
@@ -37,8 +41,50 @@ export class QTreeWidget extends QAbstractScrollArea<QTreeWidgetSignals> {
 
     selectedItems(): QTreeWidgetItem[] {
         const nativeItems = this.native.selectedItems();
-        return nativeItems.map(function(eachItem: QTreeWidgetItem) {
+        return nativeItems.map(function (eachItem: QTreeWidgetItem) {
             return new QTreeWidgetItem(eachItem);
         });
     }
+
+    /**
+     * Sets the column count of this QTreeWidget.
+     * @param columnCount The number of columns.
+     */
+    setColumnCount(columnCount: number): void {
+        this.native.setColumnCount(columnCount);
+    }
+
+    /**
+     * Sets the header label.
+     * @param label The header label.
+     */
+    setHeaderLabel(label: string): void {
+        this.native.setHeaderLabel(label);
+    }
+
+    /**
+     * Sets the header labels of the existing columns.
+     * @param labels The header labels for each column.
+     */
+    setHeaderLabels(labels: string[]): void {
+        this.native.setHeaderLabels(labels);
+    }
+
+    /**
```

Comment:
Wow! Thanks for these doc comments ! ❤️

Reply:
You're welcome. May I suggest the idea of adding doc comments wherever possible on the TS side? This might make the API more understandable (and generated docs) for users like me who have not used Qt yet. In my opinion, copying the doc descriptions of Qt could/should suffice.

Reply:
Yes, any help in this regard is welcome!